src/karmen_frontend/src/components/listings/printers-table.js

```diff
@@ -52,10 +52,16 @@ const DeletePrinterModal = ({ printer, onPrinterDelete, modal }) => {
   );
 };
 
-const PrinterSettingsModal = ({ printer, onPrinterUpdate, modal }) => {
+const PrinterSettingsModal = ({
+  printer,
+  onPrinterUpdate,
+  modal,
+  canUpdateTable,
```

Comment:
Can we use a more descriptive name for this argument? Something like `pausePrintersTableUpdates` would be great.